ta4j-core/src/main/java/org/ta4j/core/num/BigDecimalNum.java

```diff
@@ -59,7 +61,7 @@
      * @param val the string representation of the Num value
      */
     private BigDecimalNum(String val) {
-        delegate = new BigDecimal(val, MATH_CONTEXT);
```

Comment:
As mentioned in #218 this needs to be done in order for it to work, otherwise assertNumEquals throws away the precision when comparing the output. But this has no effect other then keeping the original values, so I don't think this should be a problem.